packages/bpk-component-graphic-promotion/src/accessibility-test.tsx

```diff
@@ -44,13 +42,14 @@ describe('BpkGraphicPromo accessibility tests', () => {
   it('should not have programmatically-detectable accessibility issues', async () => {
     const { container } = render(
       <BpkGraphicPromo
+        buttonText={buttonText}
         className="test"
-        tagline={tagline}
         headline={headline}
-        subheading={subheading}
-        sponsor={sponsor}
-        buttonText={buttonText}
+        invertVertically={false}
```

Comment:
Add missed `invertVertically` that TS makes us notice

Reply:
I think you're right, here in the JS we assign it a defaultValue: https://github.com/Skyscanner/backpack/blob/main/packages/bpk-component-graphic-promotion/src/BpkGraphicPromo.js#L214

In the TS we also give it a default value. 

I'd made the Props type based on just whats in the docs, but the docs and the code don't match.

I'll update the docs for it to be optional to match the code - as I think it having a default of false semantically makes sense and its the docs that are wrong.

Reply:
https://github.com/Skyscanner/backpack/pull/2535/commits/a1e654929188f7acf9bace3e84f043fd8b1735dc